### PR TITLE
Updates to the documentation - Maxim

### PR DESCRIPTION
**Pull request by maximpavliv**
Part of the updates to the documentation.

The _docs_merge_ branch is an intermediate branch to cross-review changes before opening a PR into _pytorch_dlc_.

In particular, I review the following doc files:
- _docs/course.md_
- _docs/standardDeepLabCut_UserGuide.md_
- _docs/Overviewof3D.md_
- _docs/deeplabcutlive.md_
- _docs/ModelZoo.md_
- _docs/recipes/UsingModelZooPupil.md_
- _docs/gui/PROJECT_GUI.md_
- _docs/quick-start/single_animal_quick_guide.md_
- _docs/beginner-guides/manage-project.md_
- _docs/beginner-guides/labeling.md_
- _docs/beginner-guides/Training-Evaluation.md_
- _docs/beginner-guides/video-analysis.md_
- _docs/convert_maDLC.md_
- _docs/recipes/BatchProcessing.md_
- _docs/recipes/DLCMethods.md_
- _docs/recipes/ClusteringNapari.md_
- _docs/recipes/OpenVINO.md_
- _docs/recipes/pose_cfg_file_breakdown.md_
- _docs/recipes/publishing_notebooks_into_the_DLC_main_cookbook.md_
- _docs/MISSION_AND_VALUES.md_
- _docs/Governance.md_
- _docs/roadmap.md_

Moreover, I copy the fix to build jupyter books locally from _niels/docs_. This is a **temporary** commit, it will be picked from _niels/docs_ when combining the changes.

Also, I add additional reference points to docs reviewed in _niels/docs_ to reference here, this will need to be merged.

Also, I introduce a temporary fix to `create_pretrained_project()` by forcing it to create fake metadata with TensorFlow as backend engine. This will need to be fixed once Pytorch compatibility is added to `create_pretrained_project()`.

I also remove _docs/recipes/MegaDetectorDLCLive.md_ from __toc.yml_, since the MegaDetector app is not online anymore.

Finally, I update the _.rst_ files to point to the right methods for correct doctring insertion.

### CODE REVIEW

File: docs/beginner-guides/Training-Evaluation.md [hunk 1]
```diff
@@ -40,10 +40,13 @@ After training, it's time to see how well your model performs.
 
 ### Understanding the Evaluation Results
 
-- **Performance Metrics:** DLC will assess the latest snapshot of your model, generating a `.CSV` file with performance metrics. This file is stored in the **`evaluate network`** folder within your project.
+- **Performance Metrics:** DLC will assess the latest snapshot of your model, generating a `.CSV` file with performance
+- metrics. This file is stored in the **`evaluation-results`** (for Tensorflow models) or the
+- **`evaluation-results-pytorch`** (for Pytorch models) folder within your project.
```
Comment (n-poulsen) [inline docs/beginner-guides/Training-Evaluation.md]: no new `-` needed at the start of lines or it will add bullet points

```suggestion
- **Performance Metrics:** DLC will assess the latest snapshot of your model, generating a `.CSV` file with performance 
metrics. This file is stored in the **`evaluation-results`** (for TensorFlow models) or the
**`evaluation-results-pytorch`** (for PyTorch models) folder within your project.
```

File: docs/pytorch/pytorch_config.md [hunk 2]
```diff
@@ -294,7 +294,7 @@ runner:
     save_epochs: 25  # the interval between each snapshot save  
     save_optimizer_state: false  # whether the optimizer state should be saved with the model snapshots (very little reason to set to true)
 ```
-
+[comment]: <> (todo add gpus list)
```
Comment (n-poulsen) [inline docs/pytorch/pytorch_config.md]: we should add the GPUs list here

File: docs/recipes/OpenVINO.md [hunk 3]
```diff
@@ -1,10 +1,11 @@
 # Intel OpenVINO backend
 
 DeepLabCut provides an option to run deep learning model with [OpenVINO](https://github.com/openvinotoolkit/openvino) backend.
-To enable OpenVINO in your pipeline, use `use_openvino` flag of `analyze_videos` method with one of string values indicating device:
-* "CPU" - Use CPU. This is a default value.
-* "GPU" - Use iGPU (requires OpenCL to be installed). First launch might take some time for kernels initialization.
-* "MULTI:CPU,GPU" - Use CPU and GPU simultaneously. In most cases this option provides the best efficiency.
+To enable OpenVINO in your pipeline, use `use_openvino` flag of `analyze_videos` method with one of string values
```
Comment (n-poulsen) [inline docs/recipes/OpenVINO.md]: I'd add a comment here saying OpenVINO is currently only available for the TensorFlow engine

File: docs/standardDeepLabCut_UserGuide.md [hunk 4]
```diff
@@ -11,7 +11,7 @@ To get started, you can use the GUI, or the terminal. See below.
 
 **GUI:**
 
-To begin, navigate to Aanaconda Prompt Terminal and right-click to "open as admin "(Windows), or simply launch "Terminal" (unix/MacOS) on your computer. We assume you have DeepLabCut installed (if not, see Install docs!). Next, launch your conda env (i.e., for example `conda activate DEEPLABCUT`). Then, simply run ``python -m deeplabcut``. The below functions are available to you in an easy-to-use graphical user interface. While most functionality is available, advanced users might want the additional flexibility that command line interface offers. Read more below.
+To begin, navigate to Anaconda Prompt Terminal and right-click to "open as admin "(Windows), or simply launch "Terminal" (unix/MacOS) on your computer. We assume you have DeepLabCut installed (if not, see Install docs!). Next, launch your conda env (i.e., for example `conda activate DEEPLABCUT`). Then, simply run ``python -m deeplabcut``. The below functions are available to you in an easy-to-use graphical user interface. While most functionality is available, advanced users might want the additional flexibility that command line interface offers. Read more below.
```
Comment (n-poulsen) [inline docs/standardDeepLabCut_UserGuide.md]:
```suggestion
To begin, navigate to Anaconda Prompt Terminal and right-click to "open as admin "(Windows), or simply launch "Terminal" (unix/MacOS) on your computer. We assume you have DeepLabCut installed (if not, see [install docs](how-to-install)!). Next, launch your conda env (i.e., for example `conda activate DEEPLABCUT`). Then, simply run ``python -m deeplabcut``. The below functions are available to you in an easy-to-use graphical user interface. While most functionality is available, advanced users might want the additional flexibility that command line interface offers. Read more below.
```

File: docs/standardDeepLabCut_UserGuide.md [hunk 5]
```diff
@@ -48,18 +58,43 @@ Windows users, you must input paths as: ``r'C:\Users\computername\Videos\reachin
...
+**dlc-models:** This directory contains Tensorflow models.
 
-**labeled-data:** This directory will store the frames used to create the training dataset. Frames from different videos are stored in separate subdirectories. Each frame has a filename related to the temporal index within the corresponding video, which allows the user to trace every frame back to its origin.
+**dlc-models-pytorch:** This directory contains Pytorch models.
 
-**training-datasets:**  This directory will contain the training dataset used to train the network and metadata, which contains information about how the training dataset was created.
+**dlc-models** and **dlc-models-pytorch** have a similar structure. At the top level in these directories, there are
```
Comment (n-poulsen) [inline docs/standardDeepLabCut_UserGuide.md]: I'd integrate the `**dlc-models:**` and `**dlc-models-pytorch:**` lines above into here:

```suggestion
**dlc-models** and **dlc-models-pytorch** have a similar structure: the first contains files for the TensorFlow engine while the second contains files for the PyTorch engine. At the top level in these directories, there are
```

File: docs/standardDeepLabCut_UserGuide.md [hunk 6]
```diff
@@ -48,18 +58,43 @@ Windows users, you must input paths as: ``r'C:\Users\computername\Videos\reachin
...
+curated in separation from other projects. The purpose of the subdirectories is as follows:
 
-**dlc-models:** This directory contains the subdirectories *test* and *train*, each of which holds the meta information with regard to the parameters of the feature detectors in configuration files. The configuration files are YAML files, a common human-readable data serialization language. These files can be opened and edited with standard text editors. The subdirectory *train* will store checkpoints (called snapshots in TensorFlow) during training of the model. These snapshots allow the user to reload the trained model without re-training it, or to pick-up training from a particular saved checkpoint, in case the training was interrupted.
+**dlc-models:** This directory contains Tensorflow models.
 
-**labeled-data:** This directory will store the frames used to create the training dataset. Frames from different videos are stored in separate subdirectories. Each frame has a filename related to the temporal index within the corresponding video, which allows the user to trace every frame back to its origin.
+**dlc-models-pytorch:** This directory contains Pytorch models.
```
Comment (n-poulsen) [inline docs/standardDeepLabCut_UserGuide.md]: remove as mentioned below

```suggestion
```

File: docs/standardDeepLabCut_UserGuide.md [hunk 7]
```diff
@@ -48,18 +58,43 @@ Windows users, you must input paths as: ``r'C:\Users\computername\Videos\reachin
...
+creates the symbolic links to videos in the **videos** directory. The project directory will have subdirectories:
+**dlc-models**, **dlc-models-pytorch**, **labeled-data**, **training-datasets**, and **videos**.  All the outputs
+generated during the course of a project will be stored in one of these subdirectories, thus allowing each project to be
+curated in separation from other projects. The purpose of the subdirectories is as follows:
 
-**dlc-models:** This directory contains the subdirectories *test* and *train*, each of which holds the meta information with regard to the parameters of the feature detectors in configuration files. The configuration files are YAML files, a common human-readable data serialization language. These files can be opened and edited with standard text editors. The subdirectory *train* will store checkpoints (called snapshots in TensorFlow) during training of the model. These snapshots allow the user to reload the trained model without re-training it, or to pick-up training from a particular saved checkpoint, in case the training was interrupted.
+**dlc-models:** This directory contains Tensorflow models.
```
Comment (n-poulsen) [inline docs/standardDeepLabCut_UserGuide.md]: remove as mentioned below:

```suggestion
```

File: docs/standardDeepLabCut_UserGuide.md [hunk 8]
```diff
@@ -48,18 +58,43 @@ Windows users, you must input paths as: ``r'C:\Users\computername\Videos\reachin
...
 
-**labeled-data:** This directory will store the frames used to create the training dataset. Frames from different videos are stored in separate subdirectories. Each frame has a filename related to the temporal index within the corresponding video, which allows the user to trace every frame back to its origin.
+**dlc-models-pytorch:** This directory contains Pytorch models.
 
-**training-datasets:**  This directory will contain the training dataset used to train the network and metadata, which contains information about how the training dataset was created.
+**dlc-models** and **dlc-models-pytorch** have a similar structure. At the top level in these directories, there are
+directories referring to different iterations of labels refinement (see below): **iteration-0**, **iteration-1**, etc.
```
Comment (n-poulsen) [inline docs/standardDeepLabCut_UserGuide.md]:
```suggestion
directories referring to different iterations of label refinement (see below): **iteration-0**, **iteration-1**, etc.
```

File: docs/standardDeepLabCut_UserGuide.md [hunk 9]
```diff
@@ -48,18 +58,43 @@ Windows users, you must input paths as: ``r'C:\Users\computername\Videos\reachin
...
+The refinement iterations directories store shuffle directories, each shuffle directory stores model data related to a
```
Comment (n-poulsen) [inline docs/standardDeepLabCut_UserGuide.md]:
```suggestion
The iteration directories store shuffle directories, where each shuffle directory stores model data related to a
```

**Review by n-poulsen — CHANGES_REQUESTED** [review]
This review is a WIP

**Review by n-poulsen — APPROVED** [review]
LGTM, thanks!
